commit dd7631c29284054f39f105982b0b46ad7fd4d65a
Author: John Spray <jspray@redhat.com>
Date:   Mon Apr 16 05:39:44 2018 -0400

    mgr: replace get_config_prefix with get_store_prefix
    
    The _prefix variant was only used for data-ish things,
    so we can just move it over to operate on store instead
    of config, rather than having a _prefix variant for both.
    
    Signed-off-by: John Spray <john.spray@redhat.com>

diff --git a/src/mgr/ActivePyModules.cc b/src/mgr/ActivePyModules.cc
index e61030ab686..6df6cacffb6 100644
--- a/src/mgr/ActivePyModules.cc
+++ b/src/mgr/ActivePyModules.cc
@@ -25,7 +25,6 @@
 #include "mgr/MgrContext.h"
 
 // For ::config_prefix
-#include "PyModuleRegistry.h"
 #include "PyModule.h"
 
 #include "ActivePyModules.h"
@@ -463,7 +462,7 @@ bool ActivePyModules::get_config(const std::string &module_name,
   }
 }
 
-PyObject *ActivePyModules::get_config_prefix(const std::string &module_name,
+PyObject *ActivePyModules::get_store_prefix(const std::string &module_name,
     const std::string &prefix) const
 {
   PyThreadState *tstate = PyEval_SaveThread();
@@ -479,8 +478,8 @@ PyObject *ActivePyModules::get_config_prefix(const std::string &module_name,
   
   Mutex::Locker lock(module_config.lock);
   
-  for (auto p = module_config.config.lower_bound(global_prefix);
-       p != module_config.config.end() && p->first.find(global_prefix) == 0;
+  for (auto p = store_cache.lower_bound(global_prefix);
+       p != store_cache.end() && p->first.find(global_prefix) == 0;
        ++p) {
     f.dump_string(p->first.c_str() + base_prefix.size(), p->second);
   }
diff --git a/src/mgr/ActivePyModules.h b/src/mgr/ActivePyModules.h
index 69aaa5c9f07..c10a087c4e2 100644
--- a/src/mgr/ActivePyModules.h
+++ b/src/mgr/ActivePyModules.h
@@ -77,7 +77,7 @@ public:
 
   bool get_store(const std::string &module_name,
       const std::string &key, std::string *val) const;
-  PyObject *get_config_prefix(const std::string &module_name,
+  PyObject *get_store_prefix(const std::string &module_name,
 			      const std::string &prefix) const;
   void set_store(const std::string &module_name,
       const std::string &key, const boost::optional<std::string> &val);
